build/template.phps

```diff
@@ -66,8 +66,8 @@ $opcache = (new Service($options))->handle();
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <title>OPcache statistics on <?= $opcache->getData('version', 'host'); ?></title>
-    <script src="//unpkg.com/react@16/umd/react.development.js" crossorigin></script>
-    <script src="//unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
+    <script src="//unpkg.com/react/umd/react.production.min.js" crossorigin></script>
+    <script src="//unpkg.com/react-dom/umd/react-dom.production.min.js" crossorigin></script>
```

Comment:
Removing the version boundary might potentially break the package for all the users in case of a major release with breaking changes.

Reply:
Fair enough @b1rdex - I'll look into clamping that to a major version and not specify minor/patch versions.